.jest.config.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+    clearMocks: true,
+    collectCoverage: true,
+    coverageDirectory: 'coverage',
```

Comment:
Not sure if this is needed but I've found that Jest collects coverage in a funny way if you don't specify a `collectCoverageFrom` array of glob paths. It often ends up reporting coverage on the files for which tests exist, but not on the files where tests don't exist. Here is an example PR from the SSE tool https://github.com/govuk-one-login/onboarding-self-service-experience/pull/727